### PR TITLE
Login > e2e: Add e2e test from login view

### PR DESCRIPTION
**Pull request by Andresws12**
@PVillasana @pedrojgm @JuanJosePG @JuanJosePG please, review the code and the text from the e2e test

### CODE REVIEW

File: tests/e2e/specs/utilities/01Login.selectors.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const loginView = '[data-cy="login-view"';
```
Comment (PVillasana) [inline tests/e2e/specs/utilities/01Login.selectors.ts:1]: @Andresws12 why the brackets are not closed?

File: tests/e2e/specs/01Login.spec.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+        getElem(loginSubmitButton).should('have.attr', 'disabled', 'disabled');
+
+        getElem(loginInputPassword).type('Test1.test');
+
+        getElem(loginInputPassword).should('not.have.class', isDangerClass);
+
+        getElem(loginSubmitButton).should(
+            'not.have.attr',
+            'disabled',
```
Comment (PVillasana) [inline tests/e2e/specs/01Login.spec.ts:111]: @Andresws12 What is the correct way to check for disabled elements? 

- .should('be.enabled') or this way???

File: tests/e2e/specs/utilities/01Login.selectors.ts (hunk 1 above)
Comment (Andresws12) [reply tests/e2e/specs/utilities/01Login.selectors.ts]: @PVillasana Ups, xD. Fixed

File: tests/e2e/specs/01Login.spec.ts (hunk 2 above)
Comment (Andresws12) [reply tests/e2e/specs/01Login.spec.ts]: @PVillasana Ye, sry i used this cause i am accustomed at use the old cypress version. Changed

File: tests/e2e/specs/01Login.spec.ts [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+        getElem(loginInvitationLink).click();
+
+        cy.url().should('include', '/invitation');
+
+        getElem(logoApp).click();
+
+        cy.url().should('not.include', '/invitation');
+
+        cy.log('01-03 - Login view, required inputs erros');
```
Comment (gonzalo-plaza) [inline tests/e2e/specs/01Login.spec.ts:58]: @Andresws12 fix "erros"

**Comment by gonzalo-plaza** [comment]
@Andresws12 I have a question. The button of translation must be tested in the login test or must be a test only to button of translations?

**Comment by gonzalo-plaza** [comment]
@Andresws12  For me this is okay to can be merged. @PVillasana @pedrojgm @JuanJosePG  Please review the code to merge this.

File: tests/e2e/specs/01Login.spec.ts [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+        cy.log('01-04 - Login view, check email and password');
+
+        getElem(loginInputEmail).type('test');
+
+        getElem(loginInputEmail).should('have.class', isDangerClass);
+        getElem(loginFieldEmail)
+            .find(helpSelector)
+            .should('be.visible')
+            .should('have.class', isDangerClass);
+
```
[the remark this answers is not in the document]
Comment (Andresws12) [reply tests/e2e/specs/01Login.spec.ts:88]: @gmbyana Tru, added

**Pull request merged**